examples/robust_vision_benchmark/cleverhans_attack_example/requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
```

Comment:
Why are you including a requirements.txt file here?

Reply:
The requirements.txt is only used in the Dockerfile and is part of the submission. We could make it part of the Dockerfile if you prefer to avoid confusion / reduce the number of files.

Reply:
It should be fine to leave like that, thanks for clarifying.